frontends/bmd/.env

```diff
@@ -1 +1,2 @@
 SKIP_PREFLIGHT_CHECK=true
+REACT_APP_VX_SCREEN_ORIENTATION=portrait
```

Comment:
add this to the root level .env file not here

Reply:
I'm deleting this file in a PR I just merged, but putting it in the root-level .env will make it easier to control this flag in VxDev.